deployment/terraform/dump.go

```diff
@@ -257,12 +252,36 @@ func (t *Terraform) executeDatabaseCommands(extraCommands []deployment.Cmd) erro
 func (t *Terraform) executeCommands(commands []deployment.Cmd) error {
 	for _, c := range commands {
 		mlog.Info(c.Msg)
+
+		errors := make(chan error, len(c.Clients))
+		wg := sync.WaitGroup{}
+
 		for _, client := range c.Clients {
-			mlog.Debug("Running cmd", mlog.String("cmd", c.Value))
-			if out, err := client.RunCommand(c.Value); err != nil {
-				return fmt.Errorf("failed to run cmd %q: %w %s", c.Value, err, out)
-			}
+			wg.Add(1)
+			go func() {
+				defer wg.Done()
+				mlog.Debug("Running cmd", mlog.String("cmd", c.Value))
```

Comment:
Can we also log the publicIP of the instance here to differentiate between the log lines?